packages/components/CadidatePhotos.tsx

```diff
@@ -0,0 +1,21 @@
+import { Avatar, Box, Flex, Spacer } from '@chakra-ui/react';
+
+interface CadidateCardProps {
+  candidatePhoto?: string;
+  vicePhoto?: string;
+}
+
+export default function CandidatePhotos({
+  candidatePhoto,
+  vicePhoto,
+}: CadidateCardProps) {
+  return (
+    <Box padding="1rem">
+      <Avatar boxSize="140px" src={candidatePhoto} />
+      <Spacer height="2rem" />
+      <Flex display="flex" justifyContent="center">
```

Comment:
@Guilherme-BS o componente `Flex` não precisa da propriedade `display="flex"` e se a funcionalidade do componente nesse caso for apenas centralizar algum elemento, não poderíamos usar o componente `Center`?

Reply:
troquei uma box ai esqueci de remover